### DISCUSSION TITLE
NFC修复指南

### DISCUSSION THREAD

**Discussion by dangfan** [canokeys/canokey-product#6] [General]
1. 确定电脑上没有其他程序抢占CCID
2. 进入WebConsole，选择admin，然后认证PIN通过
3. 不要刷新，直接进入APDU界面，分别执行以下两行：
00FF01000603A044000420
00FF01000903B005720300B39900
确认执行结果都是9000
如果遇到了6982，说明有其他程序干扰了执行，请关闭任何agent或换一台电脑

**Comment by dangfan** [comment]
访问 https://console.canokeys.org 在设置/Settings中验证PIN后，即可一键修复。

**Reply by 70599** [reply]
这个`一键修复`执行的和顶楼里的3步操作是一样的吗？

**Reply by dangfan** [reply]
是的，完全一致

**Comment by z4yx** [comment]
2022年下半年后的版本（1.6.2+）不再需要这些步骤
